### ISSUE TITLE
Do you plan to ever release this library?

### ISSUE THREAD

**Issue by wowkin2** [intacct/intacct-sdk-python#1]
Last commit to `dev` branch was 2 years ago. And no progress from that time.
Even links to [SDK Guides](https://developer.intacct.com/tools/sdk-python/) and [SDK Reference](https://intacct.github.io/intacct-sdk-python/)  are not working.

**Comment by Juliodonadello** [comment]
yeah, this would be awesome for consultants. And about the intacct-advantage-2016 repository, is this useful for the Data Delivery Service suscription or it is deprecated?
